srv/salt/ceph/shutdown/default.sls

```diff
@@ -0,0 +1,74 @@
+
+{% if salt['saltutil.runner']('disengage.check', cluster='ceph') == False %}
+safety is engaged:
+  salt.state:
+    - tgt: {{ salt['pillar.get']('master_minion') }}
```

Comment:
Should this be `salt['master.minion']()`?

Reply:
probably :)